index.html

```diff
@@ -16,7 +16,7 @@
                     <li><a href="index.html" class="active">About</a></li>
                     <li><a href="../pages/frontend/work.html">work</a></li>
                     <li><a href="pages/frontend/team.html">team</a></li>
-                    <li><a href="#">blog</a></li>
+                    <li><a href="blog.html">blog</a></li>
```

Comment:
This link won't be working. The directory should be similar `line 18` 

```suggestion
                    <li><a href="pages/frontend/blog.html">blog</a></li>
```

Reply:
Done